pom.xml

```diff
@@ -1202,6 +1232,11 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
         <artifactId>jul-to-slf4j</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-log4j12</artifactId>
+        <version>${slf4j.version}</version>
+      </dependency>
```

Comment:
`slf4j-log4j12` is never used (nor should it be), so this dependency is unnecessary.

(Sorry I had not noticed this in the previous round.)

Reply:
I added only those dependencies which maven suggested as required.

However, as an experiment, I removed `slf4j-log4j12` but it failed in DependencyConvergence. 

Below is the error 

```
[WARNING] 
Dependency convergence error for org.slf4j:slf4j-log4j12:1.7.10 paths to dependency are:
+-org.apache.ozone:ozone-filesystem-hadoop2:1.3.0-SNAPSHOT
  +-org.apache.hadoop:hadoop-common:2.7.3
    +-org.slf4j:slf4j-log4j12:1.7.10
and
+-org.apache.ozone:ozone-filesystem-hadoop2:1.3.0-SNAPSHOT
  +-org.apache.hadoop:hadoop-common:2.7.3
    +-org.apache.zookeeper:zookeeper:3.5.6
      +-org.slf4j:slf4j-log4j12:1.7.25
and
+-org.apache.ozone:ozone-filesystem-hadoop2:1.3.0-SNAPSHOT
  +-org.apache.hadoop:hadoop-auth:2.7.3
    +-org.slf4j:slf4j-log4j12:1.7.10

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
```